packages/devtools_shared/test/helpers/extension_test_manager_test.dart

```diff
@@ -115,7 +116,7 @@ requiresConnection: false
       expect(newerStaticExtension1Package.packageVersion, null);
       expect(
         newerStaticExtension1Package.relativePathFromExtensions,
-        'newer/static_extension_1',
+        p.join('newer', 'static_extension_1'),
```

Comment:
is there a reason this one doesn't need `p.posix.join` instead of `p.join`?

Reply:
This one is checking a file path (not a URI). I added a comment to `relativePathFromExtensions` stating that it's a path and uses the native separator, so this check must also use the native separator. The places using `p.posix` are places that are checking URIs (that always want forward slashes).